.github/workflows/md-link-checker.yml

```diff
@@ -0,0 +1,13 @@
+name: Check Markdown links
+on:
+  pull_request:
+    paths:
```

Comment:
I commented on https://github.com/cosmos/interchain-security/pull/1461 already about this workflow, let's discuss there